CookieMonster.js

```diff
@@ -3658,12 +3671,16 @@ CM.Sim.BuildingSell = function(build, basePrice, start, free, amount) {
 	return Math.ceil(price);*/
 
 	// Calculate money gains from selling buildings
+	// If noSim is set, use Game methods to compute price instead of Sim ones.
+	noSim = typeof noSim === "undefined" ? 0 : noSim;
 	var moni = 0;
+	if (amount == -1) amount = start;
+	if (!amount) amount = Game.buyBulk;
```

Comment:
I've added this to be consistent with the `Game.Object.sell` method. It allows to use the -1 value for "sell all", as it is used in `Game.buyBulk`